common/decisions/00_protestant_reformation_decision.txt

```diff
@@ -0,0 +1,60 @@
+protestant_reformation_decision = {
+	picture = {
+		reference = "gfx/interface/illustrations/decisions/decision_dynasty_house.dds"
+	}
+	desc = protestant_reformation_decision_desc
+	selection_tooltip = protestant_reformation_decision_tooltip
+	decision_group_type = major
+
+	ai_check_interval = 120
+
+	is_shown = {
+		root = {
+			OR = {
+				culture = culture:saxon
+				culture = culture:bavarian
+				culture = culture:franconian
+				culture = culture:swabian
+				culture = culture:polabian
```

Comment:
Polobian is not a germanic culture, just like for the other task, please again refer to this source:
https://ck3.paradoxwikis.com/Culture#:~:text=Original-,Central%20Germanic,-Bavarian

Any areas on the map that start with those cultures in the 1799 start date should be included in the custom region FYI

Reply:
`	is_shown = {
		root = {
			OR = {
				culture = culture:saxon
				culture = culture:bavarian
				culture = culture:franconian
				culture = culture:swabian
				culture = culture:dutch
				culture = culture:frisian
				culture = culture:german
			}
		}
	}`

Reply:
Here are the changes ^